atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -264,7 +265,9 @@ public static LockAndTimestampServices createLockAndTimestampServices(
             Environment env,
```

Comment:
Seems like we should deprecate this method call as it really shouldn't be public in the first place.